### ISSUE TITLE
Add license tag to XML

### ISSUE THREAD

**Issue by Electroid** [Electroid/PGM#496]
Allow map creators to specify a `<license>` tag for their map. Should be parsed and limited to https://creativecommons.org/choose/ for now.

**Comment by Shrman007658** [comment]
Hey! I am a newbie to Open Source. Can I take up this issue? If yes can you walk me through it? It would be very helpful! :)

**Comment by KingOfSquares** [comment]
> Hey! I am a newbie to Open Source. Can I take up this issue? If yes can you walk me through it? It would be very helpful! :)

Yes! If you join the Discord i sent you at the other issue i can help you out

**Comment by Shrman007658** [comment]
> > Hey! I am a newbie to Open Source. Can I take up this issue? If yes can you walk me through it? It would be very helpful! :)
> 
> Yes! If you join the Discord i sent you at the other issue i can help you out

Hey ! You linked me to this issue! Can I take it up? If yes can you walk me through on how to get started ? Thanks. ☺️

**Comment by KingOfSquares** [comment]
> > > Hey! I am a newbie to Open Source. Can I take up this issue? If yes can you walk me through it? It would be very helpful! :)
> > 
> > 
> > Yes! If you join the Discord i sent you at the other issue i can help you out
> 
> Hey ! You linked me to this issue! Can I take it up? If yes can you walk me through on how to get started ? Thanks. ☺️

Yes! See discord :)
